doc/Makefile.am

```diff
@@ -53,450 +53,457 @@ empty=
-     ./5.reference/tabread~-help.pd \
-     ./5.reference/tabsend-receive~-help.pd \
-     ./5.reference/tabwrite-help.pd \
-     ./5.reference/tabwrite~-help.pd \
-     ./5.reference/text-help.pd \
-     ./5.reference/text-object-help.pd \
-     ./5.reference/text-object-help.txt \
-     ./5.reference/textfile-help.pd \
-     ./5.reference/textfile.txt \
-     ./5.reference/threshold~-help.pd \
-     ./5.reference/throw~-help.pd \
-     ./5.reference/timer-help.pd \
-     ./5.reference/toggle-help.pd \
-     ./5.reference/trace-help.pd \
-     ./5.reference/trigger-help.pd \
-     ./5.reference/unpack-help.pd \
-     ./5.reference/until-help.pd \
-     ./5.reference/value-help.pd \
-     ./5.reference/vcf~-help.pd \
-     ./5.reference/vdl-help.pd \
-     ./5.reference/vline~-help.pd \
-     ./5.reference/vradio-help.pd \
-     ./5.reference/vslider-help.pd \
-     ./5.reference/vsnapshot~-help.pd \
-     ./5.reference/vu-help.pd \
-     ./5.reference/wrap~-help.pd \
-     ./5.reference/writesf~-help.pd \
-     ./5.reference/x_all_guis.pd \
-     ./6.externs/0.README.txt \
-     ./6.externs/dspobj~.c \
-     ./6.externs/makefile \
-     ./6.externs/obj1.c \
-     ./6.externs/obj2.c \
-     ./6.externs/obj3.c \
-     ./6.externs/obj4.c \
-     ./6.externs/obj5.c \
-     ./6.externs/test-dspobj~.pd \
-     ./6.externs/test-obj1.pd \
-     ./6.externs/test-obj2.pd \
-     ./6.externs/test-obj3.pd \
-     ./6.externs/test-obj4.pd \
-     ./6.externs/test-obj5.pd \
-     ./7.stuff/soundfile-tools/1.ring-mod.pd \
-     ./7.stuff/soundfile-tools/2.bandpass.pd \
-     ./7.stuff/soundfile-tools/3.phase.vocoder.pd \
-     ./7.stuff/soundfile-tools/4.looper.pd \
-     ./7.stuff/soundfile-tools/5.reverb.pd \
-     ./7.stuff/soundfile-tools/6.vocoder.pd \
-     ./7.stuff/soundfile-tools/README.txt \
-     ./7.stuff/synth/1.poly.synth.pd \
-     ./7.stuff/synth/README.txt \
-     ./7.stuff/synth/gadsr.pd \
-     ./7.stuff/synth/numset.pd \
-     ./7.stuff/synth/preset.pd \
-     ./7.stuff/synth/preset1.txt \
-     ./7.stuff/synth/preset2.txt \
-     ./7.stuff/synth/preset3.txt \
-     ./7.stuff/synth/preset4.txt \
-     ./7.stuff/synth/synthvoice.pd \
-     ./7.stuff/synth/test-gadsr.pd \
-     ./7.stuff/tools/latency.pd \
-     ./7.stuff/tools/load-meter.pd \
-     ./7.stuff/tools/miditester.pd \
-     ./7.stuff/tools/sizingtest.pd \
-     ./7.stuff/tools/testtone.pd \
-     ./7.stuff/tools/testtone16.pd \
-     ./8.topics/compander-limiter.htm \
-     ./8.topics/expr.html \
-     ./8.topics/slop-compander-patch.png \
-     ./8.topics/slop-peak-meter-graph.png \
-     ./8.topics/slop-peak-meter-patch.png \
-     ./8.topics/slop-slew-limit.png \
-     ./8.topics/slop-slew-limiting-patch.png \
-     ./8.topics/slop-tilde-1-curves.png \
-     ./8.topics/slop-tilde.htm \
-     ./Makefile.am \
-     ./sound/bell.aiff \
-     ./sound/voice.wav \
-     ./sound/voice2.wav \
+     ./1.manual/1.introduction.txt
+     ./1.manual/favicon.ico
```

Comment:
these lines all miss the line continuation backslash.

Reply:
I correct it on 9197e15